### PR TITLE
Challenge: AnasImloul submission

### PR DESCRIPTION
**Pull request by AnasImloul**
This is my second pull request. My previous pull request had some minor problems regarding the compilability of the code as it had some missing includes.

Also I just want to ask if the code will be compiled using `--std==c++17` as my code uses some of the newer version that only c++17 and newer versions support. if that's not the case. Please let me know so I can do the necessary modifications.

### CODE REVIEW

File: submissions/AnasImloul/main.cpp [hunk 1]
```diff
@@ -61,107 +59,106 @@ std::string fruits_and_vegetables[94] = {
...
-        }
+    pair<int64_t, string> cheapest = {INT64_MAX, ""};
+    for (const auto& [city, total] : total_per_city) {
+        if (total == 0) continue;
+        cheapest = min(cheapest, {total, city});
     }
 
-    output << cheapest_city << " " << std::fixed << std::setprecision(2) << cheapest_price << std::endl;
+    // print the city with the lowest price
+    output << cheapest.second
```
Comment (aboullaite) [inline submissions/AnasImloul/main.cpp:134]: Are you sure about this ?

**Review by aboullaite — CHANGES_REQUESTED** [review]

File: submissions/AnasImloul/main.cpp (hunk 1 above)
Comment (AnasImloul) [reply submissions/AnasImloul/main.cpp:134]: Thanks for the remark, I had to do it at one point of the program, but now it's not necessary. I am gonna fix it in a quick commit

Comment (aboullaite) [reply submissions/AnasImloul/main.cpp]: have you tested it ? Please do not use this as your playground 😅 we expect you to submit a fully costumised and optimised code

Comment (AnasImloul) [reply submissions/AnasImloul/main.cpp]: Yes don't worry I did test it :) I do realize that the judge server is not a made for testing for obvious reasons (test_count * 900sec per submission). The output formatting I did doesn't change the output of the program. I should have just said it's correct even though it's ugly 😅

**Review by chermehdi — APPROVED** [review]

**Pull request merged**